Kwartracker/Scenes/Wallet Page/WalletActionButtonWithLabel.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  WalletActionButtonWithLabel.swift
+//  Kwartracker
+//
+//  Created by Leah Joy Ylaya on 5/7/21.
+//
+
+import SwiftUI
+
+struct WalletActionButtonWithLabel: View {
+    @State private var buttonSize: CGFloat = 30
```

Comment:
This variable not being used?? better remove. 🤔

Reply:
forgot to use 😅